tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -935,6 +935,7 @@ def test_bobby_expect_column_values_to_be_between_auto_yes_default_profiler_conf
     version.parse(np.version.version) < version.parse("1.21.0"),
     reason="requires numpy version 1.21.0 or newer",
 )
+@probabilistic_test
```

Comment:
@NathanFarmer We do not need a second decorator call to `@probabilistic_test` -- there is one already in line 933 -- thanks!

Reply:
Nice catch! I didn't notice that line 933 got merged in from develop.